index.html

```diff
@@ -31,6 +31,192 @@ <h3>I am a Software Developer</h3>
 		 
 		</div>
 	</section>
-    
+  <section>
+		<div class="works">
+			<h2 class="recent">My Recent Works</h2>
+			<img src="Assets/backgroundcard.png" alt="">
+			<h2 class="multi-post">Multi-Post Stories</h2>
+			<p>A daily selection of privately personalized reads;
+				 no accounts or sign-ups required. has been the 
+				 industry's standard dummy text ever since the 1500s, 
+				 when an unknown printer took a standard dummy text.</p>
+
+			<ul class="buttons">
+				<li class="btn-skills">css</li>
+				<li class="btn-skills">html</li>
+				<li class="btn-skills">bootstrap</li>
+				<li class="btn-skills">Ruby</li>
+			</ul>
+
+		<ul class="btn-mid">
+			<li class="btn-project"><a href="#">See Project</a></li>
+		</ul>
+		</div>
+
+		<div class="data">
+			<h2 class="datatext">Profesional Art Printing Data</h2>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required. has been 
+				the industry's standard</p>
+
+				<ul class="btn">
+					<li class="btn-skills">html</li>
+					<li class="btn-skills">bootstrap</li>
+					<li class="btn-skills">Ruby</li>
+				</ul>
+	
+			<ul class="btn-mid">
+				<li class="btn-see-project"><a href="#">See Project</a></li>
+			</ul>
+		</div>
+	</section>
+  
+  <section>  
+  <div class="data">
+		<h2 class="datatext">Profesional Art Printing Data</h2>
+		<p>A daily selection of privately personalized reads; 
+			no accounts or sign-ups required. has been 
+			the industry's standard</p>
+
+			<ul class="btn">
+				<li class="btn-skills">html</li>
+				<li class="btn-skills">bootstrap</li>
+				<li class="btn-skills">Ruby</li>
+			</ul>
+
+		<ul class="btn-mid">
+			<li class="btn-see-project"><a href="#">See Project</a></li>
+		</ul>
+	</div>
+	</section>
+
+	<section>

```

Comment:
- [ ] All these section tags must be gone. The list of projects uses a ul tag to group them, and li per project. Please change it.